.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,90 @@
+---
+description: Report a bug.
+labels: [bug]
+name: Bug
+title: "<SCOPE>: <SUMMARY>"
+
+body:
+  - type: "checkboxes"
+
+    attributes:
+      label: "I assert that this issue is relevant for Stylix"
+      description: >-
+        If you need help or are unsure whether this is a Stylix bug, please
+        consider reading the [Documentation](https://stylix.danth.me) or asking
+        for help in a [GitHub Discussion](
+        https://github.com/danth/stylix/discussions) or the [Stylix Matrix
+        Space](https://matrix.to/#/#stylix:danth.me) before opening an issue.
+      options:
+        - label: "I assert that this is a bug and not a support request."
+          required: true
+        - label: >-
+            "I assert that this is not a [duplicate of an existing issue]
+            (https://github.com/danth/stylix/labels/bug)."
```

This here might be better:

```suggestion
            "I assert that this is not a [duplicate of an existing
            issue](https://github.com/danth/stylix/labels/bug)."
```

Are the quotes even necessary in this case:

```suggestion
            I assert that this is not a [duplicate of an existing
            issue](https://github.com/danth/stylix/labels/bug).
```